tests/Handler/CloudWatchTest.php

```diff
@@ -8,8 +8,9 @@
 use Maxbanton\Cwh\Handler\CloudWatch;
 use Monolog\Formatter\LineFormatter;
 use Monolog\Logger;
+use PHPUnit\Framework;
 
-class CloudWatchLogsTest extends \PHPUnit_Framework_TestCase
+class CloudWatchLogsTest extends Framework\TestCase
```

Comment:
I'd propose to import PHPUnit\Framework\TestCase class and then extend test class from it.